config/initializers/training_routes.rb

```diff
@@ -245,10 +232,173 @@
   OpenStruct.new(
     training_route: TRAINING_ROUTE_ENUMS[:pg_teaching_apprenticeship],
     amount: 1_000,
-    academic_cycle_id: CURRENT_ACADEMIC_CYCLE_ID,
     allocation_subjects: [
       AllocationSubjects::MODERN_LANGUAGES,
       AllocationSubjects::CLASSICS,
     ],
   ),
 ].freeze
+
+# 2023 funding rules
+
+BURSARIES_2023 = [
```

Comment:
I'm confused about this naming. When other bits of bat use a single year form, you take the lower bound of `2022 to 2023` - so `2022` not `2023`. But it would be better if we explicitly used `BURSARIES_2022_TO_2023` so there's no ambiguity.